kie-maven-plugin/src/main/java/org/kie/maven/plugin/AbstractDMNValidationAwareMojo.java

```diff
@@ -63,10 +65,21 @@
     @Parameter(property = "validateDMN", defaultValue = "VALIDATE_SCHEMA,VALIDATE_MODEL,ANALYZE_DECISION_TABLE")
     private String validateDMN;
 
+    @Parameter(property = "generateModel", defaultValue = "YES_WITHDRL") // DROOLS-5663 align kie-maven-plugin default value for generateModel configuration flag
+    private String generateModel;
+
```

Comment:
please 1 move up more. This class is called `...DMNValidationAware...`
Kindly consider move this stuff up to `AbstractKieMojo`